config/settings.yml

```diff
@@ -45,6 +45,13 @@ plugins:
     default: false
     hidden: true
     client: true
+  chat_archive_destination_topic_status:
+    type: enum
+    default: archived
```

Comment:
Would it make sense to keep it open, then direct the admin to the topic after "archiving", to decide what to do with it manually?

Reply:
Cool, can ignore this comment based on above

Reply:
I think it's best to have an automatic behaviour. By default it's changing the target topic to archived, which I think makes the most sense because you're creating a static record of chat in a point of time. However, maybe if the archive is being done to an existing topic we just skip this status change?

Reply:
> if the archive is being done to an existing topic

aye, that was the use case I was thinking.

Reply:
I will make that change then, there will be no unexpected side effects then :) Will update the site setting text to match